src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToControlGeneratedAssemblyInfo.cs

```diff
@@ -89,6 +91,175 @@ public void It_respects_opt_outs(string attributeToOptOut)
             actualInfo.Should().Equal(expectedInfo);
         }
 
+        [Fact]
+        public void It_does_not_include_source_revision_id_if_not_available1()
+        {
+            TestProject testProject = new TestProject()
+            {
+                Name = "ProjectWithSourceRevisionId",
+                IsSdkProject = true,
+                TargetFrameworks = "netcoreapp2.0",
+            };
+
+            var testAsset = _testAssetsManager.CreateTestProject(testProject);
+
+            testAsset.Restore(Log, testProject.Name);
+
+            var command = new GetValuesCommand(Log, Path.Combine(testAsset.TestRoot, testProject.Name), "netcoreapp2.0", valueName: "InformationalVersion");
+            command.Execute().Should().Pass();
+
+            command.GetValues().ShouldBeEquivalentTo(new[] { "1.0.0" });
+        }
+
+        [Fact]
+        public void It_does_not_include_source_revision_id_if_not_available2()
```

Comment:
For tests where the name only differs by a number appended to it, I'd like to see comments summarizing what the tests cover / how they are different.

Reply:
Changed the names.